packages/api/lib/ems.js

```diff
@@ -101,6 +103,22 @@ async function getExpiredS3Objects(bucket, prefix, retentionInDays) {
     .map((s3Object) => ({ Bucket: bucket, Key: s3Object.Key }));
 }
 
+async function retrievePrivateKey() {
+  const privateKeyFile = process.env.ems_privateKey || 'ems-private.pem';
+
+  const keyExists = await s3ObjectExists(
+    { Bucket: process.env.system_bucket, Key: `${process.env.stackName}/crypto/${privateKeyFile}` }
+  );
+  if (!keyExists) {
+    return Promise.reject(new Error(`${privateKeyFile} does not exist in S3 crypto directory`));
+  }
+
+  log.debug(`Reading Key: ${privateKeyFile} bucket:${process.env.system_bucket}, stack:${process.env.stackName}`);
```

Comment:
Do we need this debug statement? It's worth including the bucket and key in the exception that's thrown when the key is not found but, if it is found, this just clutters up the logs.

Reply:
Removed. Idk I took all of what was in there before.